modules/core/src/viewports/web-mercator-viewport.js

```diff
@@ -122,6 +122,8 @@ export default class WebMercatorViewport extends Viewport {
     this.bearing = bearing;
     this.altitude = altitude;
 
+    // One mercator world pixel maps to one screen pixel
+    this.pixelScale = 1;
```

Comment:
This seems a little hacky to me (i.e. worth discussion what problem this solves and if there are alternative ways of solving it.)

Reply:
`pixelScale` refers to the number of screen pixels per world space unit when projected at `pitch: 0`. We're converting between 3D to 2D with a single scaling factor, which is apparently not mathematically robust but sufficient for our use cases.